ggshield/verticals/secret/output/secret_json_output_handler.py

```diff
@@ -3,7 +3,7 @@
 from pygitguardian.client import VERSIONS
 from pygitguardian.models import PolicyBreak
 
-from ggshield.core.filter import censor_content, leak_dictionary_by_ignore_sha
+from ggshield.core.filter import leak_dictionary_by_ignore_sha
```

Comment:
Do we we still need function censor_content in ggshield/core/filter.py ? 
P-S: if you remove it, also remove the associate test ;)